rotkehlchen/chain/ethereum/modules/compound/v2/decoder.py

```diff
@@ -176,16 +181,27 @@ def _decode_borrow_and_repay(
         amount = asset_normalized_value(amount_raw, underlying_asset)
         for event in decoded_events:
             # Find the transfer event which should have come before the redeeming
-            if event.event_type == HistoryEventType.RECEIVE and event.asset.identifier == underlying_asset and event.balance.amount == amount:  # noqa: E501
+            if (
+                event.event_type == HistoryEventType.RECEIVE and
+                event.asset.identifier == underlying_asset and
+                event.balance.amount == amount and
+                event.address == compound_token.evm_address
+            ):
                 event.event_subtype = HistoryEventSubType.GENERATE_DEBT
                 event.counterparty = CPT_COMPOUND
                 event.notes = f'Borrow {amount} {underlying_asset.symbol} from compound'
-            elif event.event_type == HistoryEventType.RECEIVE and event.location_label == payer and event.asset == A_COMP and event.address == COMPTROLLER_PROXY_ADDRESS:  # noqa: E501
```

Comment:
Why are you restyling our code?

Reply:
I did for consistency, avoiding the suppression of the E501 violation, and it sounded more readable to me. Do you want that I revert this? For helping the review, I made a separate commit only with the whitespace modifications. Also for reverting it quickly if you didn't agree.